vsintegration/src/FSharp.Editor/LanguageService/Tokenizer.fs

```diff
@@ -267,7 +268,7 @@ module internal Tokenizer =
                 data.[i] <- None
                 i <- i + 1
 
-    let private dataCache = ConditionalWeakTable<DocumentId, SourceTextData>()
+    let private dataCache = ConditionalWeakTable<DocumentId, ConcurrentDictionary<string list, SourceTextData>>()
```

Comment:
This is a fix to colorization - when switching configurations, colorization information was not updated, e.g. ``#if DEBUG`` sections.